README.md

```diff
@@ -48,31 +48,31 @@ module "ocean-aks-np" {
   spotinst_account                        = var.spotinst_account
 
   # Configuration
-  ocean_cluster_name                       = "oceanClusterName"
+  ocean_cluster_name                       = "test"
   aks_region                               = "eastus"
-  controller_cluster_id                    = "controllerId"
-  aks_cluster_name                         = "aksClusterName"
-  aks_infrastructure_resource_group_name   = "aksInfrastructureResourceGroupName"
-  aks_resource_group_name                  = "resourceGroup"
+  controller_cluster_id                    = "test-123124"
+  aks_cluster_name                         = "test-cluster"
+  aks_infrastructure_resource_group_name   = "MC_TestResourceGroup_test-cluster_eastus"
+  aks_resource_group_name                  = "TestResourceGroup"
   autoscaler_is_enabled                    = true
-  autoscaler_resource_limits_max_vcpu      = 20000
-  autoscaler_resource_limits_max_memory_gib= 100000
-  autoscaler_max_scale_down_percentage     = 10
+  autoscaler_resource_limits_max_vcpu      = 750
+  autoscaler_resource_limits_max_memory_gib= 1500
+  autoscaler_max_scale_down_percentage     = 30
   autoscaler_headroom_automatic_is_enabled = true
   autoscaler_headroom_automatic_percentage = 5
   health_grace_period                      = 600
-  max_pods_per_node                        = 110
-  enable_node_public_ip                    = false
-  os_disk_size_gb                          = 128
+  max_pods_per_node                        = 30
+  enable_node_public_ip                    = true
+  os_disk_size_gb                          = 30
   os_disk_type                             = "Managed"
   os_type                                  = "Linux"
-  node_min_count                           = 0
-  node_max_count                           = 1000
-  spot_percentage                          = 100
+  node_min_count                           = 1
+  node_max_count                           = 100
+  spot_percentage                          = 50
   fallback_to_ondemand                     = true
-  availability_zones                       = [1, 2,]
-  tags                                     = { "key1": "value1", "key2": "value2" }
-  labels                                   = { "labelkey1": "labelvalue1","labelkey2": "labelvalue2"}
+  availability_zones                       = [1, 2, 3]
+  tags                                     = { "tagKey": "env", "tagValue": "staging" }
+  labels                                   = { "key": "env","value": "test"}
```

Comment:
taints, vmSizes and few other parameters missing in usage, compare and add it

Reply:
Updated all missing parameters